README.md

```diff
@@ -1,3 +1,9 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+if you would like access to the necessory envirmonet vaariable to connect to the databses locally (for developing purposes) you must follow the instruction listed below .
+you can do this by:
+1.create a file in the root directory called .env.developer
+2.creat another file in the root directory called .env.test
```

Comment:
we also need to let them know what to set here - what is the variable and where will they find the value for these variables? (database names)